easybuild/easyconfigs/t/tlparse/tlparse-0.3.37-GCCcore-13.3.0.eb

```
@@ -0,0 +1,283 @@
+    {'utf8parse-0.2.2.tar.gz': '06abde3611657adf66d383f00b093d7faecc7fa57071cce2578660c9f1010821'},
+    {'version_check-0.9.5.tar.gz': '0b928f33d975fc6ad9f86c8f283853ad26bdd5b10b7f1542aa2fa15e2289105a'},
+    {'walkdir-2.5.0.tar.gz': '29790946404f91d9c5d06f9874efddea1dc06c5efe94541a7d6863108e3a5e4b'},
+    {'wasm-bindgen-0.2.97.tar.gz': 'd15e63b4482863c109d70a7b8706c1e364eb6ea449b201a76c5b89cedcec2d5c'},
+    {'wasm-bindgen-backend-0.2.97.tar.gz': '8d36ef12e3aaca16ddd3f67922bc63e48e953f126de60bd33ccc0101ef9998cd'},
+    {'wasm-bindgen-macro-0.2.97.tar.gz': '705440e08b42d3e4b36de7d66c944be628d579796b8090bfa3471478a2260051'},
+    {'wasm-bindgen-macro-support-0.2.97.tar.gz': '98c9ae5a76e46f4deecd0f0255cc223cfa18dc9b261213b8aa0c7b36f61b3f1d'},
+    {'wasm-bindgen-shared-0.2.97.tar.gz': '6ee99da9c5ba11bd675621338ef6fa52296b76b83305e9b6e5c77d4c286d6d49'},
+    {'web-time-1.1.0.tar.gz': '5a6580f308b1fad9207618087a65c04e7a10bc77e02c8e84e9b00dd4b12fa0bb'},
+    {'winapi-0.3.9.tar.gz': '5c839a674fcd7a98952e593242ea400abe93992746761e38641405d28b00f419'},
+    {'winapi-i686-pc-windows-gnu-0.4.0.tar.gz': 'ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6'},
+    {'winapi-util-0.1.9.tar.gz': 'cf221c93e13a30d793f7645a0e7762c55d169dbb0a49671918a2319d289b10bb'},
+    {'winapi-x86_64-pc-windows-gnu-0.4.0.tar.gz': '712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f'},
+    {'windows-core-0.52.0.tar.gz': '33ab640c8d7e35bf8ba19b884ba838ceb4fba93a4e8c65a9059d08afcfc683d9'},
+    {'windows-sys-0.52.0.tar.gz': '282be5f36a8ce781fad8c8ae18fa3f9beff57ec1b52cb3de0789201425d9a33d'},
+    {'windows-sys-0.59.0.tar.gz': '1e38bc4d79ed67fd075bcc251a1c39b32a1776bbe92e5bef1f0bf1f8c531853b'},
+    {'windows-targets-0.52.6.tar.gz': '9b724f72796e036ab90c1021d4780d4d3d648aca59e491e6b98e725b84e99973'},
+    {'windows_aarch64_gnullvm-0.52.6.tar.gz': '32a4622180e7a0ec044bb555404c800bc9fd9ec262ec147edd5989ccd0c02cd3'},
+    {'windows_aarch64_msvc-0.52.6.tar.gz': '09ec2a7bb152e2252b53fa7803150007879548bc709c039df7627cabbd05d469'},
+    {'windows_i686_gnu-0.52.6.tar.gz': '8e9b5ad5ab802e97eb8e295ac6720e509ee4c243f69d781394014ebfe8bbfa0b'},
+    {'windows_i686_gnullvm-0.52.6.tar.gz': '0eee52d38c090b3caa76c563b86c3a4bd71ef1a819287c19d586d7334ae8ed66'},
+    {'windows_i686_msvc-0.52.6.tar.gz': '240948bc05c5e7c6dabba28bf89d89ffce3e303022809e73deaefe4f6ec56c66'},
+    {'windows_x86_64_gnu-0.52.6.tar.gz': '147a5c80aabfbf0c7d901cb5895d1de30ef2907eb21fbbab29ca94c5b08b1a78'},
+    {'windows_x86_64_gnullvm-0.52.6.tar.gz': '24d5b23dc417412679681396f2b49f3de8c1473deb516bd34410872eff51ed0d'},
+    {'windows_x86_64_msvc-0.52.6.tar.gz': '589f6da84c646204747d1270a2a5661ea66ed1cced2631d546fdfb155959f9ec'},
+    {'yaml-rust-0.4.5.tar.gz': '56c1936c4cc7a1c9ab21a1ebb602eb942ba868cbd44a99cb7cdc5892335e1c85'},
+]
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('maturin', '1.6.0'),
```

@Flamefire You should add `Rust` as explicit build dependency now, due to the changes in #22981
```suggestion
    ('Rust', '1.78.0'),
    ('maturin', '1.6.0'),
```